addons/residents/functions/fnc_sm_business_state_housework_enter.sqf

```diff
@@ -5,7 +5,7 @@
 private _isSleepingTime = _this call EFUNC(activities,isSleepingTime);
 
 private _house = _this getVariable ["grad_civs_home", objNull];
-_this setVariable [QGVAR(housework_time), random if (_isSleepingTime) then {GVAR(houseworkTimesNight)} else {GVAR(houseworkTimesDay)}];
```

Comment:
you *really* do like that syntax, do you.

Seriously though, I was sure I had residents in my test scenario, it should've failed right away.